#tr2:
  joint_limits:

    JointArm0:
      has_position_limits: true
      min_position: -1.0
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.2932
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointArm1:
      has_position_limits: true
      min_position: -1.4
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointArm2:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 0.25
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointArm3:
      has_position_limits: true
      min_position: -1.5708
      max_position: 1.5708
      has_velocity_limits: true
      max_velocity: 0.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0

    JointArm4:
      has_position_limits: true
      min_position: -3.0000
      max_position: 3.0000
      has_velocity_limits: true
      max_velocity: 1.4661
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 1.0
      min_effort: -1.0
